reframe/frontend/cli.py

```diff
@@ -260,8 +260,8 @@ def main():
     )
     run_options.add_argument(
         '--flex-alloc-nodes', action='store',
-        dest='flex_alloc_nodes', metavar='{all|idle|maint|NUM}', default=None,
-        help='Set strategy for the flexible node allocation (default: "idle").'
+        dest='flex_alloc_nodes', metavar='{STATE|ALL||NUM}', default=None,
+        help='Set strategy for the flexible node allocation (default: "IDLE").'
```

Comment:
I liked the lowercase here. The backends should convert them to uppercase or generally take care of the actual comparison.